web/src/components/home/subcomponents/welcome/welcome.js

```diff
@@ -1,5 +1,36 @@
+import { Box, Button, Stack } from "@mui/material";
+import welcomePic from "assets/images/teacher-demo-2-blurred-min.JPG";
+import welcomeStyles from "./welcome.styles.js";
+import FeatureFlags from "infra/feature_flags";
+
 function Welcome() {
-  return <div>Welcome</div>;
+  return (
+    <Stack sx={welcomeStyles.sectionLayout}>
+      <Stack justifyContent="center" sx={welcomeStyles.titleBox}>
+        <Box>
+          <Box sx={welcomeStyles.subtitle}>Discover your rhythm at</Box>
+          <Box sx={welcomeStyles.title}>Do Your Own Swing</Box>
+        </Box>
+        <Box>
+          <Box>
+            West Coast Swing classes and social dancing in San Jose, CA.
+          </Box>
+          <Box>Thursday Nights @ Studio M Ballroom</Box>
+        </Box>
+        {FeatureFlags.showStartHereTab && (
+          <Button sx={welcomeStyles.startHereButton}>
```

Comment:
You should just be able to add `color="primary"` instead of setting the background color in the styles

Reply:
I have a lot more going on for that style than the color

Reply:
that doesn't work - the button is transparent and that only sets the text color

Reply:
You probably also need `variant="contained"`

prefer to rely on the default behavior than custom styling for easier re-use